packages/salesforcedx-vscode-soql/package.nls.json

```diff
@@ -1,4 +1,6 @@
 {
+  "configuration_title": "Salesforce SOQL Configuration",
   "soqlCustom_soql": "SOQL Builder",
-  "soql_builder_open_new": "SFDX: Create Query in SOQL Builder"
+  "soql_builder_open_new": "SFDX: Create Query in SOQL Builder",
+  "soql_editor_remote_checks_flag": "Validate SOQL editor queries against your Org. When enabled, the SOQL editor calls the remote API to validate the query on your default Org environment and provide better syntax checks."
```

Comment:
```suggestion
  "soql_editor_remote_checks_flag": "Validate SOQL editor queries against your org. When enabled, the SOQL editor calls the remote API to validate the query on your default org and provide better syntax checks."
```

Reply:
Also, couldn't figure out how to put my comment closer to the .mov. Can we all cap SOQL "Experimental: SOQL Editor Remote Checks" - but still want to make sure I understand the context for 'editor' here as well

Reply:
I would make this description a little more compact. Maybe something like "Enable SOQL query validation using the REST API and display errors".

Reply:
I don't think we need to mention better syntax checks. Is the fact that it's REST API important? Validate SOQL queries and identify errors. (?)

Reply:
The reason I want to put "REST API" is that it will tell the customers that the checks happens on the server side. That way they would (hopefully) know that this validation wouldn't work if they are offline or having an auth issue and/or not connected to an org.